core/common/src/main/java/alluxio/util/io/FileUtils.java

```diff
@@ -295,7 +298,11 @@ public static void createFile(String filePath) throws IOException {
    * @param path path of the directory to create
    */
   public static void createDir(String path) throws IOException {
-    new File(path).mkdirs();
+    try {
+      Files.createDirectories(Paths.get(path));
+    } catch (AccessDeniedException e) {
+      LOG.info("Fail to create folder " + path + "caused by AccessDeniedException");
```

Comment:
I add a throw IOException in catch block and run test, there are three errors:
`  SpaceReserverTest.reserveCorrectAmountsOfSpace:77 » IO Fail to create folder /...
  SpaceReserverTest.testHighWatermark:166 » IO Fail to create folder /a
  SpaceReserverTest.testLowWatermark:115 » IO Fail to create folder /a`